@types/ipfs/index.d.ts

```diff
@@ -1,3 +1,5 @@
+import { DAGImportOptions, DAGImportResult } from 'kubo-rpc-client';
```

Comment:
iiuc we dont use types from kubo-rcp-client for all other commands,  but have local definitions in @types/ipfs/index.d.ts.
maybe follow existing pattern and do the same?